doc/ui-testing-with-maestro.md

```diff
@@ -1,43 +1,27 @@
+
 # UI testing with Maestro
 
+Simple has different product flavours and it is important to verify the integrity of the UI across these variants. As Maestro offers a single binary tool that works anywhere with declarative yet robust syntax and has a quick learning curve, we use it for UI testing in Simple.
```

Comment:
We are not doing UI testing because of product flavors. So, we might want to change this terminology. @Honey14 can you help Noyal out on this.